docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 3
+title: Quickstart
+---
+
+# Quickstart
+
+Follow these steps for a quick start:
+
+1. Install Chainloop. This quick snippet will get you through the process:
+    ```bash
+    curl -sfL https://docs.chainloop.dev/install.sh | bash -s
+    ```
+    Refer to [these instructions](/getting-started/installation) for more installation options.
+
+2. Authenticate to the Control Plane via OIDC by running:
+    ```bash
+    chainloop auth login
+    ```
+    This will create a personal token that you will use in operations not related to attestations.
+
+3. Create a new Chainloop workflow.
+    ```bash
+    chainloop wf create --name mywf --project myproject
+    ```
+    Chainloop workflows represent any CI or process you might want to attest.
+    Check [this doc](/getting-started/workflow-definition)
+    for a complete explanation of Workflows and Contracts.
+    You might also want to check our [contract reference](/reference/operator/contract).
+
+4. Create a restricted token for further operations:
+    ```bash
+    export CHAINLOOP_TOKEN=$(chainloop org api-token create --name test-api-token -o json | jq -r ".[].jwt")
+    ```
+    CHAINLOOP_TOKEN environment variable is commonly used from CI/CD scenarios, where a personal token is not available.
+    Tokens have narrower permissions, ensuring that they can only perform the operations they are granted to.
+    More information in [API tokens](/reference/operator/api-tokens#api-tokens).
+
+5. Sign a payload and push the attestation to Chainloop
+    ```bash
+    chainloop att init --workflow-name mywf
+    chainloop att add --value "ghcr.io/chainloop-dev/chainloop/control-plane:latest"
```

Comment:
have you checked if you can add remote files? I think that would be great to be implemented (in another task). We already have some code for this to retrieve the contracts locally or remotely, and it could enable a quickstart that points to one of our sample SBOMs for example

Reply:
I've tested it. They are interpreted as STRING types. It's a great topic for a new feature.